docs/docs/pricing/README.md

```diff
@@ -84,63 +99,42 @@ To discuss an Enterprise plan, please contact the Pipedream Sales Team [using th
-
-Your workflow's [memory settings](/workflows/settings/#memory) also impact the number of invocations you're charged for each workflow execution. [Read more here](#how-does-workflow-memory-affect-billable-invocations).
+Adding additional memory capacity to workflows will increase credit usage in intervals of 256 megabytes. For example, doubling the memory of a workflow from {{ $site.themeConfig.base_credits_price.memory }} to {{ $site.themeConfig.base_credits_price.memory * 2 }} will double the cost of credits in the same execution time.
 
 #### Scenarios
 
-::: details Webhook triggered workflow
+::: details A standard workflow that runs 5 times with one second per run
 
-*1* invocation is incurred per HTTP webhook. The HTTP endpoint is *not* considered a source.
+Only 1 credit is incurred, because the workflow ran for a total of 5 seconds (5 runs at 1 second each = 5 total seconds of compute time)
 
 :::
 
-::: details Scheduled workflow
-
-*1* invocation is incurred per new event emitted by the source.
-*1* invocation is incurred per new event processed by the workflow.
+::: details Developing a workflow with test events in the Pipedream workflow builder
 
-Example: a schedule triggered workflow is configured to run every 15 minutes. Two invocations are incurred every 15 minutes. One from the timer source emitting an event, and the other from the workflow processing the event.
-
-```
-2 invocations * 15 minutes * 4 times per hour * 24 hours in a day = 2,880 daily invocations
-```
+0 credits are incurred. Developing your workflows is free. Execution time used to develop a workflow in the builder does not count towards your credit usage.
 
 :::
 
-::: details App webhook powered source triggered workflow
-
-*1* invocation is incurred per new event emitted by the source.
-*1* invocation is incurred per new event processed by the workflow.
+::: details An active standard workflow that isn't triggered at all in a billing period
 
-Example: an _Slack New Message in Channel (Instant)_ source receives a webhook from Slack when a new message is received in a channel. Two total invocations are incurred, one from the Slack source emitting the message event, the other from the workflow processing the event.
+0 credits are incurred. You only pay for execution time.
 
 :::
 
-::: details App polling source triggered workflow
-
-*1* invocation is incurred per new event emitted by the source.
-*1* invocation is incurred per new event processed by the workflow.
+::: details An upgraded workflow with 512 megabytes of memory running for 5 seconds
 
-Example: a _Twitter Search Tweets_ source checks finds a new tweet published. Two total invocations are incurred, one from the source emitting the tweet event, the other for the workflow processing the event.
+2 credit are incurred, because the workflow ran for a total of 5 seconds (5 runs at 1 second each = 5 total seconds of compute time)
```

Comment:
It looks like the "(5 runs at 1 second each = 5 total seconds of compute time)" may not apply to this example. But it's correct that this would be two credits.